### DISCUSSION TITLE
Chrome on Android - js example

### DISCUSSION THREAD

**Discussion by mikepotter21** [ggerganov/ggwave#40] [Q&A]
Firstly, congrats on your work. Loving it.
I have tried the example at https://ggwave-js.ggerganov.com/ on Chrome on Samsung 8, 9 and latest Google phone, but can not get the sound to play. This is only on Android phones using Chrome. On Chrome on iPhone, Chrome desktop (PC and OS) all fine. I have then tried Chrome Developer Tools connecting to Android with Chrome to hunt the issue, but works as script must me running through the desktop browser, although trying to debug mobile.
Samsung browser is fine on same Androids, so assuming is Chrome specific issue. 
Have reset browsers, checked settings
Chrome version: 91.0.4472.101 on the device currently testing.
Android 9 SM G950F Build/PPR1.180610.011.
Has anyone else experienced this or found a work around?
Thanks again
Mike

**Comment by ggerganov** [comment]
Hi Mike,

I have also observed recently that on Pixel 5 using latest Chrome, receiving ggwave messages does not work at all. Seemed like there was some sort of strong filtering or pre-processing being applied to the captured audio. I haven't investigated in details though. Does the receiving part work for you?

I wasn't aware that the playback option also does not work.
Thanks for reporting this.

cc @rgerganov

**Reply by mikepotter21** [reply]
No prob. Wish I had a fix for you or could contribute more :)
The receiving part is crashing the browser. But chrome for me on any other device apart from Android OS is fine. Just the Chrome for Android where I have experienced both send and receive issues. Both can be replicated from the ggwav-js  example page. However on using Chrome devtools for debug, it works as it must be using the desktop engine somewhere in the mix.
Thanks
